inspirehep/modules/theme/templates/inspirehep_theme/format/record/Base_HTML_brief.tpl

```diff
@@ -0,0 +1,49 @@
+{#
+# This file is part of INSPIRE.
+# Copyright (C) 2015, 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{% from "inspirehep_theme/format/record/Base_HTML_general_macros.tpl" import render_record_title, render_record_authors, record_abstract, record_arxiv, record_report_numbers, record_cite_modal with context %}
+
+{% macro brief_format(record) %}
+{% set pub_info = record|publication_info %}
+<div class="row">
+  <div class="col-md-12">
+    <div id="panel-default-brief" class="panel panel-default" >
+      <div class="panel-body" >
+        {% block panel_body scoped %}
+        <div class="col-md-12">
+          <h4 class="custom-h">
+          {% block title_body scoped %}{% endblock %}
+          </h4>
+          <div class="row">
+            <div class="col-md-12 record-brief-details">
```

Comment:
`record-brief-details` class is applied here and below, can you check what it does and it might make sense to add that class to the container of both rows.